src/averageAges.js

```diff
@@ -15,11 +15,13 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filterMen = !century
+    ? people.filter(person => person.sex === 'm')
+    : people.filter(person => person.sex === 'm'
+      && Math.ceil(person.died / 100) === century);
+
+  return filterMen.reduce((sum, per) =>
+    sum + (per.died - per.born), 0) / filterMen.length;
 }
```

Comment:
```suggestion
  const filterMen = !century
    ? people.filter(person => person.sex === 'm')
    : people.filter(person => (
      person.sex === 'm' && Math.ceil(person.died / 100) === century));

  return filterMen.reduce((sum, per) =>
    sum + (per.died - per.born), 0) / filterMen.length;
}
```

I think this will be more readable if you put all checks on the same line
You should also use round brackets if you move your shortened return statement in arrow function to a newline (that's from checklist)